index.html

```diff
@@ -21,131 +21,180 @@
+              <ul class="multi-post-ul">
+                <li><img src="img/Tagcss.png"></li>
+                <li><img src="img/Taghtml.png"></li>
+                <li><img src="img/Tagbootstrap.png"></li>
+                <li><img src="img/Tagruby.png"></li>
+              </ul>
+              <button class="multi-post-nav">See project</button>
+            </div> 
+          </div>
+
       </div>
-      <div class="profesional-art-container-2">
-        <h2 class="profesional-art-h2-2">Profesional Art<br>Printing Data</h2>
-        <p class="profesional-art-p-2">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
-        <ul class="profesional-art-ul-2">
-          <li><img src="img/Taghtml2.png"></li>
-          <li><img src="img/Tagbootstrap2.png"></li>
-          <li><img src="img/Tagruby2.png"></li>
-        </ul> 
-        <button class="profesional-art-nav-2">See project</button>
-      </div>
-      <div class="profesional-art-container-3">
-        <h2 class="profesional-art-h2-3">Profesional Art<br>Printing Data</h2>
-        <p class="profesional-art-p-3">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
-        <ul class="profesional-art-ul-3">
-          <li><img src="img/Taghtml2.png"></li>
-          <li><img src="img/Tagbootstrap2.png"></li>
-          <li><img src="img/Tagruby2.png"></li>
-        </ul>
-        <button class="profesional-art-nav-3">See project</button> 
-      </div>
-      <div class="profesional-art-container-4">
-        <h2 class="profesional-art-h2-4">Profesional Art<br>Printing Data</h2>
-        <p class="profesional-art-p-4">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
-        <ul class="profesional-art-ul-4">
-          <li><img src="img/Taghtml2.png"></li>
-          <li><img src="img/Tagbootstrap2.png"></li>
-          <li><img src="img/Tagruby2.png"></li>
-        </ul> 
-        <button class="profesional-art-nav-4">See project</button>
+      
+      <div class="cards-container">
+
+        <div class="profesional-art-container">
+          <button class="profesional-art-nav">See project</button>
+        </div>
+  
+        <div class="profesional-art-container-Data-Dashboard">
+          <h2 class="profesional-art-h2">Data Dashboard<br>Healthcare</h2>
+          <p class="profesional-art-p">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+          <ul class="profesional-art-ul">
+            <li><img src="img/Taghtml2.png"></li>
+            <li><img src="img/Tagbootstrap2.png"></li>
+            <li><img src="img/Tagruby2.png"></li>
+          </ul> 
+        </div>
```

Comment:
- [ ] Impressive work you did here on the card 👏 . However, note that the white background with the `button` should display on over on each card, and the card should return back to the blurry version when not hoverd.
- This functiioality was given as a guide on the template, and i suggest you kindly do so for all cards involved. Good job 👍 

## Card expected hover design:
![image](https://github.com/bohaz/First-mobile-project/assets/50990925/7c6e28cb-a5d8-49fc-b5f4-3a73de76723f)

## When not hovered:
![image](https://github.com/bohaz/First-mobile-project/assets/50990925/d7a8602e-afdd-4bde-9ef6-5ca720c147ce)